zeppelin-server/src/main/java/org/apache/zeppelin/socket/Message.java

```diff
@@ -52,6 +52,9 @@
     NEW_NOTE, // [c-s] create new notebook
     DEL_NOTE, // [c-s] delete notebook
               // @param id note id
+    CLONE_NOTE, // [c-s] clone new notebook
```

Comment:
Yes, I was thinking of fixing it in a different PR.

בתאריך יום ו׳, 21 באוג׳ 2015, 01:19 מאת Felix Cheung <
notifications@github.com>:

> In zeppelin-server/src/main/java/org/apache/zeppelin/socket/Message.java
> <https://github.com/apache/incubator-zeppelin/pull/223#discussion_r37587769>
> :
>
> > @@ -52,6 +52,9 @@
> >      NEW_NOTE, // [c-s] create new notebook
> >      DEL_NOTE, // [c-s] delete notebook
> >                // @param id note id
> > +    CLONE_NOTE, // [c-s] clone new notebook
>
> is this a different PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-zeppelin/pull/223/files#r37587769>.
>